src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterExtensions.cs

```diff
@@ -47,7 +47,7 @@ public static void WriteMetricsCollection(this PrometheusExporter exporter, Stre
             foreach (var metric in exporter.Metrics)
             {
                 var builder = new PrometheusMetricBuilder()
-                    .WithName(metric.Name)
+                    .WithName(metric.Meter.Name + metric.Name)
```

Comment:
I guess this is not the correct way?

Reply:
The SDK requirement is documented https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view. In particular, search for "The SDK SHOULD use the following logic to determine how to process Measurements made with an Instrument".